textproc/scdoc/Portfile

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                scdoc
+version             1.9.4
+categories          textproc
+license             MIT
+maintainers         herrbischoff.com:marcel
```

Comment:
MacPorts would ideally like your GitHub handle to be specified here as well; doing so allows you to be automatically notified of any proposed changes to your port via GitHub.

Additionally, would you be interested in having this port be `openmaintainer`?

Reply:
Sure. I just thought it would be useful to have someone "responsible" for it.
Or do you mean it to be like so?

```
maintainers         @herrbischoff herrbischoff.com:marcel \
                    openmaintainer
```

Reply:
Done.

Reply:
For specifying your GitHub handle, could you please group them in braces (i.e.  `{@herrbischoff herrbischoff.com:marcel}`?

Reply:
Already did, found the suggestion to do that in the docs... :)